contracts/src/Epoch.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.15;
+
+import "./NodeRegistry.sol";
+
+
+contract EpochManager {
+    /**
+     * STATE *
+     */
+    uint256 public epoch;
+
+    uint256 public currentEpochEndStampMs;
+
+    uint256 public currentCommitteeSize;
+
+    uint256 public maxCommitteeSize;
+
+    uint256 public epochDurationMs;
+
+    NodeRegistry public nodeRegistry;
+    bool private initialized;
+    uint256 private readyToChange;
+
+    struct CommitteeMember {
+        string publicKey;
+        string primaryAddress;
+        string workerAddress;
+        string workerMempool;
```

Comment:
isn't this a list of worker addresses?

Reply:
It can be but we only have 1 worker per node for now so I stored it like this. Ill refactor to future proof it

Reply:
refactored